### PR TITLE
0920 fpki tools

### PR DESCRIPTION
**Pull request by claytonjbarnette**
FPKIMA SITE map is hard to find GSA/idmanagement.gov#392

Added link to FPKIMA  page to side nav
@JBPayne007

### CODE REVIEW

**Review by JBPayne007 — APPROVED** [review]

File: _fpki/5_fpki_tools.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ subnav:
       href: ../tools/cct/
     - text: Certificate Profile Conformance Tool
       href: ../tools/cpct/
+    - text: FPKIMA Sitemap
+      href: https://www.idmanagement.gov/governance/fpkiaudit/#audit-information-for-the-fpki-management-authority
```
Comment (idmken) [inline _fpki/5_fpki_tools.md:21]: This type of href doesn't work.

File: _fpki/5_fpki_tools.md [hunk 2]
```diff
@@ -36,3 +38,4 @@ crt.sh | [External](https://crt.sh/){:target="_blank"}{:rel="noopener noreferrer
 OpenSC | [Github (External)](https://github.com/OpenSC/OpenSC/wiki){:target="_blank"}{:rel="noopener noreferrer"} | OpenSC provides a set of libraries and utilities to work with smart cards. Use OpenSC versions of 0.20.0 or greater to reduce website authentication errors related to TLS 1.3. | System Engineers
 NIST 85B (800-73-4) Test Tool | [NIST (External)](https://csrc.nist.gov/projects/nist-personal-identity-verification-program/software-downloads){:target="_blank"}{:rel="noopener noreferrer"} | Used to pull deep PIV contents when integrating PIV with various infrastructure components. | System Engineers; FPKI Card Issuers
 Treasury PIV Authentication Tester (TLS 1.3) | [External](https://caia-dev.treasury.gov/pagination-spa){:target="_blank"}{:rel="noopener noreferrer"} | A tool from the U.S. Department of the Treasury, Bureau of the Fiscal Service used to test PIV authentication. Click on "log in" and follow the prompts for PIV. This tool will identify if your agency is impacted by TLS 1.3 break and inspect for Trusted Internet Connection (TIC). | System Engineers; PIV Holders
+
```
Comment (idmken) [inline _fpki/5_fpki_tools.md:41]: The easiest way to add something is to the table. You should be part of the FICAM Coders team and can create branches in the repo.

**Review by idmken — CHANGES_REQUESTED** [review]

**Comment by idmken** [comment]
@claytonjbarnette are you going to update these?

File: _fpki/5_fpki_tools.md (hunk 2 above)
Comment (claytonjbarnette) [reply _fpki/5_fpki_tools.md:41]: 👍

File: _fpki/5_fpki_tools.md (hunk 1 above)
Comment (claytonjbarnette) [reply _fpki/5_fpki_tools.md:21]: Understood. 👍

File: _fpki/5_fpki_tools.md (hunk 2 above)
Comment (claytonjbarnette) [reply _fpki/5_fpki_tools.md:41]: @idmken This was my first thought and set up for this, but the target result kept moving, the result was me trying to add a link to the nav as request... I will circle back around on this.

**Comment by idmken** [comment]
I don't see any activity on this PR and it's open for almost Two months. If this is still needed, open a new branch in this repo and provide the update.

**Pull request closed without merging**